From c5c126aa3fb9582a149ea44c88aa72eccc374ce2 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Thu, 16 Jan 2020 06:01:20 +1300
Subject: Fix broken tests

These fail due to implicit dependency on paths in the upstream
workspace laid-out github repo, and thus topologically impossible
in a published crate.
---
 src/stream/tests.rs | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/stream/tests.rs b/src/stream/tests.rs
index f811012..bc629b5 100644
--- a/src/stream/tests.rs
+++ b/src/stream/tests.rs
@@ -174,6 +174,7 @@ fn test_incomplete_frame() {
     );
 }
 
+#[cfg(broken_test)]
 #[test]
 fn test_cli_compatibility() {
     let input = include_bytes!("../../assets/example.txt.zst");
@@ -189,7 +190,7 @@ fn test_cli_compatibility() {
     );
 }
 
-#[cfg(feature = "legacy")]
+#[cfg(all(broken_test,feature = "legacy"))]
 #[test]
 fn test_legacy() {
     use std::fs;
@@ -232,6 +233,7 @@ fn test_empty() {
     }
 }
 
+#[cfg(broken_test)]
 #[test]
 fn test_ll_source() {
     // Where could I find some long text?...
-- 
2.24.0

